models/Asset/Thumbnail/ImageThumbnailTrait.php

```diff
@@ -189,13 +189,19 @@ private function readDimensionsFromFile(): array
                 $localFile = $this->getLocalFile();
                 if (null !== $localFile) {
                     if ($imageInfo = @getimagesize($localFile)) {
-                        $this->getAsset()->getDao()->addToThumbnailCache(
-                            $this->getConfig()->getName(),
-                            basename($pathReference['storagePath']),
-                            filesize($localFile),
-                            $imageInfo[0],
-                            $imageInfo[1]
-                        );
+                        $dimensions = [
```

Comment:
@blankse what is the reason for defining `$dimensions`? 
Using `$imageInfo[0]` ... should be sufficient, right?

Reply:
@brusch For the return value of readDimensionsFromFile. Instead its always an empty array right?

Reply:
Like it was before https://github.com/pimcore/pimcore/pull/12461/files#diff-b48489e5be12fb4a4ec4f716c03e4690a799c39a87430cbb1a435d7badebf97cL193

Reply:
🤦 thanks for clarifying 😊